app/subscriber/src/store/hooks/lookup/useLookup.ts

```diff
@@ -156,13 +158,27 @@ export const useLookup = (): [ILookupState, ILookupController] => {
           'lookup',
         );
       },
+      getTonePools: async () => {
```

Comment:
This isn't really something we need.  We're not currently allowing users to choose someone else's tonepool.  Also the way you have currently implemented this could result in hundreds of records returned in this request when at most the user would only need their own tonepool and the default tonepool.

Reply:
When reviewing the `LookupController` I see it is also currently configured to return all tonepools.  This isn't ideal and should be changed to at most, only those which are public.

Reply:
Removed to avoid retriving hundred of records.